### PR TITLE
updated https in place of http

### LINKED ISSUE #175: https in place of http
At least in the bottom link « This question is issued from http://dsp.stackexchange.com«.

### PR DESCRIPTION
**Pull request by ritikjaiswal019**
Updated to https in place of http

This fixes #175

### CODE REVIEW

File: sotoki/sotoki.py [hunk 1]
```diff
@@ -1677,11 +1677,12 @@ def run():
     if tag_depth != -1 and tag_depth <= 0:
         sys.exit("--tag-depth should be a positive integer")
     domain = arguments["<domain>"]
-    if re.match("^https?://", domain):
-        url = domain
-        domain = re.sub("^https?://", "", domain).split("/")[0]
-    else:
-        url = "http://" + domain
+    if re.match("^https?://",domain):
```
Comment (rgaudin) [inline sotoki/sotoki.py:1680]: please respect the code formatting ; ie. space after a comma.

File: sotoki/sotoki.py [hunk 2]
```diff
@@ -1677,11 +1677,12 @@ def run():
...
+        domain = re.sub("^https://","",domain)
+    elif re.match("^http?://",domain):
```
Comment (rgaudin) [inline sotoki/sotoki.py:1682]: this is wrong and useless. remove the elif block

File: sotoki/sotoki.py [hunk 3]
```diff
@@ -1677,11 +1677,12 @@ def run():
     if tag_depth != -1 and tag_depth <= 0:
         sys.exit("--tag-depth should be a positive integer")
     domain = arguments["<domain>"]
-    if re.match("^https?://", domain):
-        url = domain
-        domain = re.sub("^https?://", "", domain).split("/")[0]
-    else:
-        url = "http://" + domain
+    if re.match("^https?://",domain):
+        domain = re.sub("^https://","",domain)
```
Comment (rgaudin) [inline sotoki/sotoki.py:1681]: this won't work on http:// ones… use `re.sub("^https?://", "", domain).split("/")[0]`

File: sotoki/sotoki.py [hunk 4]
```diff
@@ -1677,11 +1677,12 @@ def run():
...
+    elif re.match("^http?://",domain):
+        domain = re.sub("^http://","",domain)
+    url = "https://" + domain
```
Comment (rgaudin) [inline sotoki/sotoki.py:1684]: you can't do that before removing the path

File: sotoki/sotoki.py [hunk 5]
```diff
@@ -1677,11 +1677,12 @@ def run():
...
+    domain = domain.split("/")[0]
```
Comment (rgaudin) [inline sotoki/sotoki.py:1685]: Should be done before, especially now that the elif is to be dropped

**Review by rgaudin — CHANGES_REQUESTED** [review]
Thank you for you PR but you need to be more careful about what you submit. The answer to this is already in a comment on the other PR.
Having a review comment on every single line is not a good PR. I spent more time reviewing this than it would require to fix it directly.

File: sotoki/sotoki.py (hunk 1 above)
Comment (ritikjaiswal019) [reply sotoki/sotoki.py:1680]: I will surely be keeping this in mind from the next time.

File: sotoki/sotoki.py (hunk 2 above)
Comment (ritikjaiswal019) [reply sotoki/sotoki.py]: Done

**Review by rgaudin — APPROVED** [review]
Thanks for the fix !

**Comment by ritikjaiswal019** [comment]
Thank you for your guidance @rgaudin. From the next time, I will be keeping in mind the changes you suggested.

**Pull request merged**